nest-city-account/Dockerfile

```
@@ -29,6 +36,15 @@ CMD [ "npm", "run", "start:debug" ]
 
 # production
 FROM app-base AS prod
+# When copying from build, it copies symlink in node_modules to ../../openapi-clients and expects the original directory
+# to be present.
+# TODO: Find a way how to use --install-links flag in npm install to avoid this.
+RUN mkdir -p /home/node/openapi-clients && chown -R node:node /home/node/openapi-clients
```

(optional) Don't understand the comment, but I don't really care that much :smile:. Just prefer different formatting
```suggestion
RUN mkdir -p /home/node/openapi-clients \
 && chown -R node:node /home/node/openapi-clients
```

---

I tried to fix the underlying problem but spend 1 hour+ to do that, note this is not something I came up with here, it is copied from Nest FE Backend and just applied here.

At least I improved the comment to be more descriptive https://github.com/bratislava/konto.bratislava.sk/pull/2353/commits/bab606371294c773f3e12d5842be05727f067442

---

Fixed the formatting.